pysimplesoap/client.py

```diff
@@ -37,6 +37,7 @@
                      get_local_name, get_namespace_prefix, TYPE_MAP, urlsplit, \
                      iteritems
 from .wsse import UsernameToken
+from urllib3.util import make_headers
```

Comment:
I'd like to keep this in alignment with the maintainers, as far as I can tell, they put the minimum assumptions on the imports used. in other words, this should work in environment without urllib3. When this is done, I'd like you create a Pull request into the main pysimplesoap repo.